templates/_macros/split-list.jinja

```diff
@@ -7,7 +7,11 @@
 # title: split list title text
 # list_items: array of content objects for list
 #   [{ 'title': string, 'description': string }]
-{% macro split_list(variant=1, title='', list_items=[{}]) -%}
+{% macro split_list(variant=1, title='', subtitle='', list_items=[{}]) -%}
```

Comment:
I guess it would make sense to call it `title` and `description`, similar to title/description inside the list?